frontend/src/Components/Link/Link.js

```diff
@@ -47,11 +47,7 @@ class Link extends Component {
         el = 'a';
         linkProps.href = to;
         linkProps.target = target || '_self';
-      } else if (to.startsWith(`${window.Radarr.urlBase}/`)) {
-        el = RouterLink;
-        linkProps.to = to;
-        linkProps.target = target;
-      } else {
+        } else {
```

Comment:
fix the indentation here

Reply:
you're slacking missed one (other file, line 56); lint caught to...i suppose it helps to actually lint it